### ISSUE TITLE
Inline comments breaking if comment begins with @ symbol

### ISSUE THREAD

**Issue by callmenick** [babel/babel-sublime#412]
If I begin an inline comment with an `@` symbol, the comment gets some highlighting applied on the first line…

Example:

```javascript
// note: var foo = 'foo'
// var foo = 'foo'
// Lorem ipsum dolor sit amet, consectetur adipisicing elit.

// @note: var foo = 'foo'
// var foo = 'foo'
// Lorem ipsum dolor sit amet, consectetur adipisicing elit.
```

Result:

<img width="763" alt="Screen Shot 2021-05-27 at 2 04 48 PM" src="https://user-images.githubusercontent.com/2130125/119875222-8655ce00-bef4-11eb-9ee9-08f137131732.png">

**Comment by Thom1729** [comment]
This is a feature, not a bug (part of the Flow typing support). It is arguably not a very good feature and I've gotten more complaints than compliments over it. The Flow support is due for a revision in any case; I might remove the special comments entirely, or move them out into a separate option.

**Comment by callmenick** [comment]
Ah interesting. Thanks for the reply :) Look forward to the updates.

**Comment by Slapbox** [comment]
Please please make this feature off by default. We're rolling back to the prior version. We're in the middle of a major refactor and comments beginning with `@` are everywhere, and our code is a nightmare to read that way. It might be better if the comments weren't shown with the full color/opacity.

Also, the highlighting can be broken by a URL appearing in the comment, and the comment color can vary based on the exact location of the comment:

![image](https://user-images.githubusercontent.com/6835891/120937530-f6760800-c6db-11eb-8399-590cf3d7df74.png)

**Comment by Thom1729** [comment]
I've removed Flow comment directives in v11.1.0-beta.1.

**Comment by Thom1729** [comment]
Fixed in v11.1.0.
